frontend/src/Editor/LeftSidebar/index.jsx

```diff
@@ -46,39 +48,143 @@ export const LeftSidebar = forwardRef((props, ref) => {
     updateOnPageLoadEvents,
     apps,
     clonePage,
+    setEditorMarginLeft,
     isVersionReleased,
     setReleasedVersionPopupState,
   } = props;
   const queryPanelHeight = usePanelHeight();
-  const [selectedSidebarItem, setSelectedSidebarItem] = useState();
+  const [selectedSidebarItem, setSelectedSidebarItem] = useState(localStorage.getItem('selectedSidebarItem'));
   const [showLeaveDialog, setShowLeaveDialog] = useState(false);
   const [showDataSourceManagerModal, toggleDataSourceManagerModal] = useState(false);
   const [popoverContentHeight, setPopoverContentHeight] = useState(queryPanelHeight);
+  const [pinned, setPinned] = useState(!!localStorage.getItem('selectedSidebarItem'));
+  const [errorLogs, setErrorLogs] = React.useState([]);
+  const [errorHistory, setErrorHistory] = React.useState({ appLevel: [], pageLevel: [] });
+  const [unReadErrorCount, setUnReadErrorCount] = React.useState({ read: 0, unread: 0 });
```

Comment:
```suggestion
  const [errorLogs, setErrorLogs] = useState([]);
  const [errorHistory, setErrorHistory] = useState({ appLevel: [], pageLevel: [] });
  const [unReadErrorCount, setUnReadErrorCount] = useState({ read: 0, unread: 0 });
```

`useState` is already imported and we can use it directly. We can maintain the consistency